lib/pipeline.js

```diff
@@ -1384,6 +1384,7 @@ class PipelineModel extends BaseModel {
             .then(async scmRepo => {
                 this.scmRepo = scmRepo;
                 this.name = scmRepo.name;
+                this.private = scmRepo.private;
```

Comment:
Do we need to add this field in [data-schema](https://github.com/screwdriver-cd/data-schema/blob/master/models/pipeline.js?)

Reply:
yeah it's in another pr